app/src/main/java/in/tosc/digitaloceanapp/adapters/DataCenterAdapter.java

```diff
@@ -40,31 +36,57 @@ public DataCenterViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(DataCenterViewHolder holder, int position) {
         this.postion = holder.getAdapterPosition();
-        String thisCountry = countriesList.get(position).getCity();
-        int url = countriesList.get(position).getId();
-
-        holder.countryName.setText(thisCountry);
-        Picasso.with(context).load(url).resize(425,220).into(holder.img);
+        String thisRegion = regions.getRegions().get(position).getName();
+        holder.countryName.setText(thisRegion);
+        if(thisRegion.contains("New York"))
```

Comment:
I couldn't find any way to use switch case with String.contains() method. Please tell how it can be done.
<img width="731" alt="screen shot 2017-05-28 at 9 01 21 pm" src="https://cloud.githubusercontent.com/assets/19621727/26529960/0e92e4be-43e9-11e7-8b75-7172b05da428.png">